cli/jobs/parallel/1a_oj_sales_prediction/src/partition_data/partition_data.yml

```diff
@@ -0,0 +1,29 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
```

Comment:
Seems this worth to be a built-in component in public registry.

Reply:
Yeah, maybe we could consider registry later. Now it has potential risk on bugs :P